specification/httpx/src/main/scripts/org/kaazing/specification/httpx/extended/client.sends.message.between.opening.and.extended.handshake/request.rpt

```diff
@@ -38,21 +38,12 @@ read header "Date" /.+/
 
 write [0x82 0x85] ${writeMask}
 write option mask ${writeMask}
-write "Hello"
+write "World"
```

Comment:
I see what happens in HTTP decoding when Hello is sent as method. Can you not just send "Hello World"